runtime/gc_realtime/ProcessorInfo.cpp

```diff
@@ -95,15 +95,15 @@ double
 MM_ProcessorInfo::readFrequency()
 {
 	double freq = 0;
-#if (defined(LINUX) && (defined(J9X86) || defined(J9HAMMER) || defined(LINUXPPC)))
+#if (defined(LINUX) && (defined(J9X86) || defined(J9HAMMER) || defined(LINUXPPC) || defined(J9RISCV64)))
```

Comment:
I don't know how it works for RISC-V. See comment below. If we will need it for Metronome we should extra test this API

Reply:
Agreed and removed for now.